tests/testClassifier.py

```diff
@@ -29,6 +29,7 @@
 import lsst.ctrl.stats.records as recordslib
 from lsst.ctrl.stats.reader import Reader
 from lsst.ctrl.stats.classifier import Classifier
+import eups
```

Comment:
Please use `lsst.utils.getPackageDir()` rather than EUPS for locating a product.